crates/router/src/core/utils.rs

```diff
@@ -400,3 +396,61 @@ pub async fn construct_upload_file_router_data<'a>(
     };
     Ok(router_data)
 }
+
+#[instrument(skip_all)]
+pub async fn construct_retrieve_file_router_data<'a>(
+    state: &'a AppState,
+    merchant_account: &storage::MerchantAccount,
+    file_metadata: &storage_models::file::FileMetadata,
+    connector_id: &str,
+) -> RouterResult<types::RetrieveFileRouterData> {
+    let db = &*state.store;
+    let connector_label = file_metadata
+        .connector_label
+        .clone()
+        .ok_or(errors::ApiErrorResponse::InternalServerError)
+        .into_report()
+        .attach_printable("Missing connector label")?;
+    let merchant_connector_account = helpers::get_merchant_connector_account(
+        db,
+        merchant_account.merchant_id.as_str(),
+        &connector_label,
+        None,
+    )
+    .await?;
+    let auth_type: types::ConnectorAuthType = merchant_connector_account
+        .get_connector_account_details()
+        .parse_value("ConnectorAuthType")
+        .change_context(errors::ApiErrorResponse::InternalServerError)?;
+    let router_data = types::RouterData {
+        flow: PhantomData,
+        merchant_id: merchant_account.merchant_id.clone(),
+        connector: connector_id.to_string(),
+        payment_id: String::new(),
```

Comment:
Instead of `String::new()` Can we have a const like `unused_pay_id_in_dispute_flow` or any other name as so that it is easier to understand when it is logged or used anywhere.